doc/sphinx/package_service_mapping.json

```diff
@@ -509,7 +509,21 @@
     "namespaces": [
       "azure.mgmt.machinelearningcompute"
     ],
-    "service_name": "Machine Learning Compute"
+    "service_name": "Machine Learning"
```

Comment:
I think you can let that be Machine Learning Compute since it is a service by itself. did that in js too